index.html

```diff
@@ -259,6 +259,26 @@ <h6 class="description">Hello I’m a software developer! I can help you build a
       </section>
     </section>
   </div>
-       </section>
+</section>
+<footer class="form-cont">
+  <form action="https://formspree.io/f/mrgdkyla" method="post">
+    <h2 class="h2">I'm always<br>interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    <input type="text" id="name" placeholder="Full name" required maxlength="30" name="full name"><div class="line"></div>
+    <input type="email" placeholder="Email address" required><div class="line" name="email"></div>
+    <textarea name="text" maxlength="500" required>Enter text here</textarea>
+    <button type="submit" class="get-in see1">Get in touch</button>
+  </form>
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly make sure you add all the interaction required on the buttons. I noticed that it is still missing the `active or pressed` button interaction.
![Screenshot from 2022-10-04 22-15-00](https://user-images.githubusercontent.com/83042742/193917451-a70c7aa9-96db-4a2a-800a-0103b077ea47.png)